src/Constant/Priority.php

```diff
@@ -0,0 +1,14 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\ApiFilter\Constant;
+
+final class Priority
+{
+    public const HIGHEST = 1705;
```

Comment:
Where did you get those numbers 🤔?

Reply:
Those numbers are [Tribonacci numbers](https://cs.wikipedia.org/wiki/Fibonacciho_posloupnost#Tribonacciho_%C4%8D%C3%ADsla).

It's similar to Fibonacci but they are more _wide_ so there are more possibilities to _inject_ between them if you need to. (_And few first numbers are removed because it is not necessary to have so small diversity._)

```
Fibonacci
1, 89, 144, 233, 377, 610, 987

Tribonacci
1, 81, 149, 274, 504, 927, 1705
```

But if you want, I can change them for more common `Fibonacci` but I think it really doesn't matter really :)

Reply:
Well I don't care than much, I was just curious where they come from ;-)